### PR TITLE
Seo

### PR DESCRIPTION
**Pull request by dimarconicola**
##### Description

Basic SEO

##### Checklist

- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Testing

Checking it with https://app.neilpatel.com/ and possibly Google Lighthouse

##### Refers/Fixes

#24

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# https://www.robotstxt.org/robotstxt.html
+# robots.txt for https://pie.dexwallet.io/
```
Comment (Alexintosh) [inline public/robots.txt]: wrong domain

File: public/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <lastmod>2020-01-12T19:55:45Z</lastmod>
+  </url>
+    <url>
+    <loc>https://pie.dexwallet.io/Discord/</loc>
+    <changefreq>Monthly</changefreq>
+    <priority>1</priority>
+    <lastmod>2020-01-12T19:55:45Z</lastmod>
+  </url>
+    <url>
+    <loc>https://pie.dexwallet.io/InvestmentDetail/</loc>
```
Comment (Alexintosh) [inline public/sitemap.xml]: wrong domain

File: src/utils/config.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
+    DEBUG: process.env.NODE_ENV === 'development',
+    titlePrivacy: 'Privacy',
+    titlePro: 'Dexwallet PRO',
```
Comment (Alexintosh) [inline src/utils/config.js]: wat

File: src/utils/config.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    titleTerms: 'Terms & Conditions',
+    titleWhitelabel: 'Dexwallet Whitelabel',
```
Comment (Alexintosh) [inline src/utils/config.js]: wat

File: src/utils/config.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  
+    siteName: 'Pie - Top investment for everyone',
+    author: 'Pie Network',
+    description:
+      'The richest people in the World have gained 10% every year for the past 10 years. We think it is fair for you to make some real money too.',
+    DiscordDescription: 'Redirecting to Pie Network Discord Channel',
+    siteUrl: 'https://www.pie.dexwallet.io/',
+    image: 'https://www.pie.dexwallet.io/img/preview.jpg',
+  
+    twitter: '@Dexwallet',
```
Comment (Alexintosh) [inline src/utils/config.js:14]: wat

File: src/utils/config.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    titleWhitelabel: 'Dexwallet Whitelabel',
+    titleBuy: 'Buy Cryptocurrencies',
+    titleDiscord: 'Dexwallet Discord Channel',
+  
+    siteName: 'Pie - Top investment for everyone',
+    author: 'Pie Network',
+    description:
+      'The richest people in the World have gained 10% every year for the past 10 years. We think it is fair for you to make some real money too.',
+    DiscordDescription: 'Redirecting to Pie Network Discord Channel',
+    siteUrl: 'https://www.pie.dexwallet.io/',
```
Comment (Alexintosh) [inline src/utils/config.js]: wrong domain

File: src/utils/config.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    image: 'https://www.pie.dexwallet.io/img/preview.jpg',
```
Comment (Alexintosh) [inline src/utils/config.js]: does it exist?

**Review by Alexintosh — CHANGES_REQUESTED** [review]

File: src/utils/config.js (hunk 7 above)
Comment (dimarconicola) [reply src/utils/config.js]: it does

**Pull request merged**